Library/Formula/fossil.rb

```diff
@@ -1,16 +1,8 @@
 class Fossil < Formula
   homepage "https://www.fossil-scm.org/"
   head "https://www.fossil-scm.org/", :using => :fossil
-  url "https://www.fossil-scm.org/download/fossil-src-20150223162734.tar.gz"
-  sha256 "c00f9e1487530adb921df730baf57e1aa23767a0afb60b865e5842bb50782942"
-  version "1.31"
-
-  bottle do
-    cellar :any
-    sha256 "0008984514cf0956aca9371492c8ff779ab224378d620f9a97e204516b899f0c" => :yosemite
-    sha256 "ac24fa102938c7f533e9d9a28c1d622fafb2ddc54e3e2a28d2fe630a95fa9a3b" => :mavericks
-    sha256 "0e136626581d82192245d1373aeaeaa5a4a71369f08ecf8d008713e7ad5ae529" => :mountain_lion
-  end
```

Comment:
Please leave the `bottle` block as it is. It will be updated when merging. Thanks.